package.json

```diff
@@ -110,9 +108,6 @@
     "webpack-dev-server": "^3.1.11",
     "webpack-merge": "^4.2.1"
   },
-  "peerDependencies": {
-    "nightwatch": "^0.9.12"
-  },
   "optionalDependencies": {
     "wdio-mocha-framework": "^0.5.11"
   },
```

Comment:
Nightwatch is still sneaking around in a few places, do we want to remove the devDependency on it?

Reply:
I cannot believe I missed that! 🤦‍♀️